server/channels/app/notification.go

```diff
@@ -877,6 +877,22 @@ func (a *App) SendNotifications(c request.CTX, post *model.Post, team *model.Tea
 		mlog.String("post_id", post.Id),
 	)
 
+	go func() {
+		for id, reason := range mentions.Mentions {
+			user, ok := profileMap[id]
+			if !ok {
+				continue
+			}
+			if user.IsGuest() {
+				if reason == KeywordMention {
+					a.Srv().telemetryService.SendTelemetryForFeature("guest", false, "post_triggered_guest_notification", map[string]any{})
```

Comment:
Since KeywordMention means "The post contains an at-mention for the user" - should we rename this event to smth. that is closer to that meaning?

Reply:
it can be an at-mention or a keyword that triggers the mention, both things are considered a mention. but we can change it to something like "post_mentioned_guest" or something similar